### PR TITLE
#7389 dropdown remove

### LINKED ISSUE #7389: remove dropdown and redirect after click - Create selection
image
- the dropdown is now redundant. You can get to the mass mint by clicking in the selection. Could we please remove it and redirect the user after only clicking the "create" 
thanks 

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #7389
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI

Screen Shot 2023-10-16 at 9 15 41 PM

### CODE REVIEW

**Comment by prury** [comment]
dark mode showing white mode image: 
image

**Comment by prachi00** [comment]
@exezbcz can you provide the svg for dark mode ?

**Comment by exezbcz** [comment]
> @exezbcz can you provide the svg for dark mode ? 

It's in figma already

**Comment by prachi00** [comment]
@prury 
clicking create on mobile won't close the modal:

what do you mean by this? this dropdown seems to get closed after clicking

Screen Shot 2023-10-17 at 8 24 27 PM

**Comment by prachi00** [comment]
@prury done

File: components/create/CreateLanding.vue [hunk 1]
```diff
@@ -25,6 +25,26 @@
...
+        <button
+          v-if="urlPrefix === 'ahk'"
+          :class="{ 'pack-box-waifu p-0': !isMobile }"
+          class="pack-box"
+          @click="router.push({ path: `/${urlPrefix}/waifu` })">
+          <div :class="[!isMobile ? 'pack-content-waifu' : 'pack-content']">
+            <img
+              v-if="!isMobile"
+              :src="isDarkMode ? '/waifu-dark.svg' : '/waifu.svg'"
+              class="svg-one" />
```
Comment (preschian) [inline components/create/CreateLanding.vue:37]:
```suggestion
              class="svg-one" alt="Create Waifu NFT" />
```

**Review by preschian — APPROVED** [review]

File: components/create/CreateLanding.vue [hunk 2]
```diff
@@ -25,6 +25,26 @@
             </div>
           </div>
         </button>
+        <button
+          v-if="urlPrefix === 'ahk'"
+          :class="{ 'pack-box-waifu p-0': !isMobile }"
+          class="pack-box"
+          @click="router.push({ path: `/${urlPrefix}/waifu` })">
```
Comment (roiLeo) [inline components/create/CreateLanding.vue:32]: please use `NuxtLink`

File: components/create/CreateLanding.vue [hunk 3]
```diff
@@ -131,6 +153,18 @@ const gotoPathAfterLogin = (path: RawLocation) => {
...
+  &-box-waifu {
+    width: 17rem;
+  }
+
+  &-content-waifu {
+    text-align: center;
+
+    &-text {
+      margin-top: 2rem;
+    }
```
Comment (roiLeo) [inline components/create/CreateLanding.vue:165]: `mt-6`

**Comment by exezbcz** [comment]
mising label on the top 

image



Could you please center the heading and subheading vertically so that the colored rectangles are evenly spaced?

image

File: components/create/CreateLanding.vue (hunk 3 above)
Comment (prachi00) [reply components/create/CreateLanding.vue:165]: Screen Shot 2023-10-18 at 9 53 02 PM
so this class is already there, I didn't add it, its suppose to be margin-top 0 for mobile thats why theres a separate class I think

**Comment by prachi00** [comment]
@exezbcz solved

**Review by roiLeo — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Review by prury — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 3.74 USD/DOT ~ 8.021 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x3182f4763e0ffec3b5e61aa3e306b84ba07f9bd4a09321fd7513c6b758f094a5](https://polkadot.subscan.io/extrinsic/0x3182f4763e0ffec3b5e61aa3e306b84ba07f9bd4a09321fd7513c6b758f094a5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**